### ISSUE TITLE
AppImage 2.5.4 not accessible to Linux screen reader

### ISSUE THREAD

**Issue by fwiep** [keepassxreboot/keepassxc#4853]
## Overview
The AppImage release 2.5.0 was relatively good accessible for my screen reader (Orca). Having updated to release 2.5.4, the application is no longer accessible. My screen reader stays completely silent.

## Steps to Reproduce
1. Enable screen reader (often by <kbd>Alt</kbd> + <kbd>Super</kbd> + <kbd>S</kbd>, or `orca --replace` on the command line)
2. Launch the AppImage release 2.5.4
3. Not that not even the window title is spoken

## Expected Behavior
The application should be accessible, as it was in previous release (at least 2.5.0).

## Actual Behavior
The application is not accessible; the screen reader stays completely silent.

## Context
KeePassXC - Version 2.5.4
Revision: dcca5aa
Distribution: AppImage

Qt 5.12.7
Debugging mode is disabled.

Operating system: Debian GNU/Linux 10 (buster)
CPU architecture: x86_64
Kernel: linux 4.19.0-9-amd64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
 libgcrypt 1.8.1

Operating System: Linux
Desktop Env: Gnome
Windowing System: X11

**Comment by droidmonkey** [comment]
We didn't change anything regarding the screen reader capability between those versions. Are you sure orca has permission to read that window?

**Comment by fwiep** [comment]
Thanks for the quick response.

It seems the inaccessible behaviour was due to my own Orca-experiments (building from `master` on a daily basis).

All versions op KeepassXC are (still) accessible: 2.5.0, 2.5.1, 2.5.2, 2.5.3 and 2.5.4.

Sorry for the noise.
FWieP
